proto/juno/clock/v1/genesis.proto

```diff
@@ -13,17 +13,22 @@ message GenesisState {
     (gogoproto.nullable) = false,
     (gogoproto.jsontag) = "params,omitempty"
   ];
-}
-
-// Params defines the set of module parameters.
-message Params {
   // contract_addresses stores the list of executable contracts to be ticked on every block.
-  repeated string contract_addresses = 1 [    
+  repeated string contract_addresses = 2 [    
```

Comment:
note: this will require a module store Migration handler (to move from previous params store -> new genesis store.

Governance uploaded contracts should allow for higher gas limits.

ex: standard contract uploads only get the `Params.standard_contract_gas_limit`, but if it is uploaded via gov we set it to a higher amount (ex: Params.privileged_contract_gas_limit) that would be 100m or 2bn or something

Reply:
No longer needed as no gov proposals have been made